src/Book.java

```diff
@@ -15,7 +17,7 @@ public String getBookTitle() {
     }
 
     public Author getNameOfAuthor() {
```

Comment:
```suggestion
    public Author getAuthor() {
```
геттеры должны совпадать с названием полей